server/public/model/config.go

```diff
@@ -1763,6 +1765,10 @@ func (s *FileSettings) SetDefaults(isUpdate bool) {
 		s.AmazonS3UploadPartSizeBytes = NewPointer(int64(FileSettingsDefaultS3UploadPartSizeBytes))
 	}
 
+	if s.AmazonS3StorageClass == nil || *s.AmazonS3StorageClass == "" {
+		s.AmazonS3StorageClass = NewPointer("STANDARD")
```

Comment:
This should be an empty string right?

Reply:
That is correct.
I have pushed a commit that fixes this.